ddtrace/internal/telemetry/writer.py

```diff
@@ -86,12 +88,13 @@ def send_event(self, request):
         except Exception:
             log.debug("failed to send telemetry to the Datadog Agent at %s.", self.url, exc_info=True)
         finally:
-            conn.close()
+            if conn:
+                conn.close()
```

Comment:
is this a fix for a bug in telemetry writer? it seems like it is unrelated to this PR? If so, can you open as it's own PR?